code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -154,7 +154,12 @@
 			break
 
 /obj/structure/closet/proc/open(mob/living/user)
-	if(opened || !can_open(user))
+	if(!can_open(user))
+		return
+	return openFully()
+
+/obj/structure/closet/proc/openFully()

```

Comment:
There are things that will break if this ever gets called directly.
Namely the stealth box, aswell as the stealth box from the implant.

Reply:
Ah damnnit you're right. Fuck. Do you think there's any way I can get the effect without just making a typecache?

Reply:
should? be fixed